e2e-test/java/io/cdap/plugin/slt/actions/SLTActions.java

```diff
@@ -0,0 +1,111 @@
+
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.SeleniumDriver;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.plugin.slt.locators.SLTLocators;
+import io.cdap.plugin.slt.utils.CDAPUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.io.IOException;
+
+/**
+ * SLT Actions.
+ */
+public class SLTActions {
+    public static SLTLocators sltLocators = new SLTLocators();
+    private static WebDriver CDFDriver;
+    private static ElementHelper elementHelper = new ElementHelper();
+
+    static {
+        CDFDriver = SeleniumDriver.getDriver();
+        sltLocators = SeleniumHelper.getPropertiesLocators(SLTLocators.class);
+
+    }
+    public static void name_fill(String name) {
+        sltLocators.name.sendKeys(name);
+    }
+    public void required_config(String gcpProjectId,
+                                String gcsDataPath,
+                                String massTransferId,
+                                String sapJcoLibGcsPath,
+                                String jcoclienthost,
+                                String jcoclient,
+                                String jcoclientsysnr,
+                                String jcoclientuser,
+                                String jcoclientpasswd) throws IOException {
+        elementHelper.replaceElementValue(sltLocators.gcpProjectId,
+                CDAPUtils.getPluginProp(gcpProjectId));
+        elementHelper.sendKeys(sltLocators.gcsDataPath, CDAPUtils.getPluginProp(gcsDataPath));
+        elementHelper.sendKeys(sltLocators.massTransferId, massTransferId);
+        elementHelper.sendKeys(sltLocators.sapJcoLibGcsPath, CDAPUtils.getPluginProp(sapJcoLibGcsPath));
+        elementHelper.sendKeys(sltLocators.jcoclienthost, CDAPUtils.getPluginProp(jcoclienthost));
+        elementHelper.sendKeys(sltLocators.jcoclient, CDAPUtils.getPluginProp(jcoclient));
+        elementHelper.sendKeys(sltLocators.jcoclientsysnr,CDAPUtils.getPluginProp(jcoclientsysnr));
+        elementHelper.sendKeys(sltLocators.jcoclientuser, CDAPUtils.getPluginProp(jcoclientuser));
+        elementHelper.sendKeys(sltLocators.jcoclientpasswd, CDAPUtils.getPluginProp(jcoclientpasswd));
+    }
+    public void sltplugin_click() {
+        elementHelper.clickOnElement(sltLocators.sltplugin);
+    }
+
+    public String rowError_color() {
+        return elementHelper.getElementCssProperty(SLTLocators.rowError, "border-color");
+    }
+
+    public void next_click() {
+        elementHelper.clickOnElement(sltLocators.next);
+    }
+
+    public void select_table(String tableName) {
+        elementHelper.clickOnElement(sltLocators.table(tableName));
+    }
+
+    public void datasetName_fill (String stagingBucket) {
+        elementHelper.sendKeys(sltLocators.datasetName, stagingBucket);
+    }
+
+    public void deployPipelineClick() {
+        elementHelper.clickOnElement(sltLocators.deployPipeline);
+    }
+
+    public void startPipeline() {
```

Comment:
clickStartPipelineButton

Check all method names. Should start with action(i.e click , enter , select, get etc)

Reply:
Updated as per method action.